.code-samples.meilisearch.yaml

```diff
@@ -29,12 +29,18 @@ date_guide_filterable_attributes_1: |-
   await client.Index("games").UpdateFilterableAttributesAsync(new string[] { "release_timestamp" });
 date_guide_filter_1: |-
   var filters = new SearchQuery() { Filter = "release_timestamp >= 1514761200 AND release_timestamp < 1672527600" };
-  SearchResult<Game> games = await client.Index("games").SearchAsync<Movie>("", filters);
+  SearchResult<Game> games = await client.Index("games").SearchAsync<Game>("", filters);
 date_guide_sortable_attributes_1: |-
   await client.Index("games").UpdateSortableAttributesAsync(new string[] { "release_timestamp" });
 date_guide_sort_1: |-
   SearchQuery sort = new SearchQuery() { Sort = new string[] { "release_timestamp:desc" }};
-  await client.Index("games").SearchAsync<Meteorite>("", sort);
+  await client.Index("games").SearchAsync<Game>("", sort);
+filtering_guide_nested_1: |-
+  var filters = new SearchQuery() { Filter = "rating.users >= 90" };
+  SearchResult<Movie> movies = await client.Index("movie_ratings").SearchAsync<Movie>("thriller", filters);
```

Comment:
```suggestion
  SearchResult<MovieRatings> movies = await client.Index("movie_ratings").SearchAsync<MovieRatings>("thriller", filters);
```

Reply:
There is no `MovieRatings` class, and even if you call your index `movie_ratings`, it does not mean that your class is too.

Reply:
Since the `index`  is different  (`movie_ratings` instead of `movie`) the document structure is different as well. So `<Movie>` does not work

Reply:
```suggestion
  SearchResult< MovieRatings > movies = await client.Index("movie_ratings").SearchAsync< MovieRatings >("thriller", filters);
```